openmmtools/integrators.py

```diff
@@ -999,31 +1304,41 @@ class LangevinSplittingIntegrator(ThermostatedIntegrator):
         - g-BAOAB with solvent-solute splitting, K_r=K_p=2:
             splitting="V0 V1 R R O R R V1 R R O R R V1 V0"
 
+    Attributes
+    ----------
+    _kinetic_energy : str
+        This is 0.5*m*v*v by default, and is the expression used for the kinetic energy
+
     References
     ----------
+    [Nilmeier, et al. 2011] Nonequilibrium candidate Monte Carlo is an efficient tool for equilibrium simulation
     [Leimkuhler and Matthews, 2015] Molecular dynamics: with deterministic and stochastic numerical methods, Chapter 7
     """
 
     def __init__(self,
+                 alchemical_functions,
                  splitting="V R O R V",
```

Comment:
It might be helpful to change the default splitting to include an alchemical example using H. On related note, the splitting docstring should be updated to include H as an option.